tests/fixtures/SwaggerServiceTest/tmp_data_get_user_request_without_request_class.json

```diff
@@ -0,0 +1,99 @@
+{
+    "openapi": "3.1.0",
+    "servers": [
+        {
+            "url": "http:\/\/localhost"
+        }
+    ],
+    "paths": {
+        "/users/{id}/assign-role/{role-id}": {
+            "get": {
+                "tags": [
+                    "users"
+                ],
+                "consumes": [],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "in": "path",
+                        "name": "id",
+                        "description": "",
+                        "required": true,
+                        "schema": {
+                            "type": "string"
+                        }
+                    },
+                    {
+                        "in": "path",
+                        "name": "role-id",
+                        "description": "",
+                        "required": true,
+                        "schema": {
+                            "type": "string"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
```

Comment:
The second change is the `description` in the `response` key. because of `null` result of `getConcreteRequest` method, the default status text will be printed.
https://github.com/RonasIT/laravel-swagger/blob/07aab91443233d767184b7f2d73f60610380e4bc/src/Services/SwaggerService.php#L764-L768